Tests/IntegrationTests/SharedNodeTypesPackage/SharedNodeTypesPackage

```diff
@@ -0,0 +1 @@
+../Packages/Application/Neos.Neos.Ui/Tests/IntegrationTests/SharedNodeTypesPackage
```

Comment:
What's this?

Reply:
That is maybe the reason for the  note in the next point... did not see that :/
Just wondered that the symlink could not be created. Therefore (rm - rf DistributionPackages/Neos.TestNodeTypes)